mix.exs

```diff
@@ -65,72 +65,87 @@ defmodule DotCom.Mixfile do
   # Specifies your project dependencies.
   #
   # Type `mix help deps` for examples and options.
+  #
+  # You can check the status of each dependency by running `mix hex.outdated`.
   defp deps do
     [
-      {:absinthe_client, "~> 0.1.0"},
-      {:benchfella, "~> 0.3", [only: :dev]},
-      {:bypass, "~> 1.0", [only: :test]},
-      {:castore, "~> 0.1.11"},
-      {:con_cache, "~> 0.12.0"},
+      {:absinthe_client, "0.1.0"},
+      {:benchfella, "0.3.5", [only: :dev]},
+      # latest version 2.1.0; cannot be upgraded because of errors when we use * in the url for tests
+      # rather than fixing tests and upgrading, we should convert the tests to use mox
+      {:bypass, "1.0.0", [only: :test]},
+      # latest version 1.0.5; cannot upgrade because of server_sent_event_stage expects castore < 1
```

Comment:
Some more detail for this note: technically we can change that (as [`:server_sent_event_stage`](https://github.com/mbta/server_sent_event_stage) is ours)!

Looking at other libraries that have more flexible requirements for `:castore`, I found that `:absinthe_client` and `:phoenix` list ">= 0.0.0" while "~> 0.1 or ~> 1.0" is used for `:elixir_make`, `:finch`, `:mint`, and `:redix`. So we could change `:server_sent_event_stage` to use that. 

However that might not be enough to help Dotcom since `:tesla` also expects "~> 0.1". That library is required by `:web_driver_client` which is required by `:wallaby`, which we use for parsing the DOM tree for one task.

Reply:
Yeah, I was going to note that we own that project in the follow-up ticket. Maybe we evaluate whether or not we still need it? There might be community libraries that do the same thing.

Reply:
I more meant...we can't upgrade it without more work. Technically, we could upgrade anything in the list given the right amount of effort.